### ISSUE TITLE
TRAIN_CLS is inf

### ISSUE THREAD

**Issue by saicoco** [qjadud1994/OCR_Detector#7]
Thanks for your job! When I train it on icdar15, the loss of classification is inf as following:

```
STEP : 513	TRAIN_TOTAL_LOSS : inf	TRAIN_LOC_LOSS : 0.09856237	TRAIN_CLS_LOSS : inf
```
How should I do? I think the loss about classification is focal loss, pow(x) will be inf if x is inf

**Comment by saicoco** [comment]
Tensorflow version fix： tf.log(pred+eps) can solve this problem
